extensions/observability-devservices/deployment/src/main/java/io/quarkus/observability/deployment/ObservabilityDevServiceProcessor.java

```diff
@@ -152,19 +152,22 @@ public void startContainers(LaunchModeBuildItem launchMode,
             }
 
             if (firstStart.computeIfAbsent(devId, x -> true)) {
-                Runnable closeTask = () -> {
-                    DevServicesResultBuildItem.RunningDevService current = devServices.get(devId);
-                    if (current != null) {
-                        try {
-                            current.close();
-                        } catch (Throwable t) {
-                            log.errorf("Failed to stop %s container", devId, t);
+                Runnable closeTask = new Runnable() {
```

Comment:
Just leave the lambda?

Reply:
Nope. This is a good practice in the extensions to reduce the number of classes at runtime. We don't care about it on user code but extensions should.